build_projects/dotnet-host-build/DebTargets.cs

```diff
@@ -57,6 +73,17 @@ public static BuildTargetResult GenerateSharedFrameworkDeb(BuildTargetContext c)
             var inputRoot = c.BuildContext.Get<string>("SharedFrameworkPublishRoot");
             var debFile = c.BuildContext.Get<string>("SharedFrameworkInstallerFile");
             var objRoot = Path.Combine(Dirs.Output, "obj", "debian", "sharedframework");
+            var debianConfigFile = Path.Combine(Dirs.RepoRoot,
+                "packaging", "deb-package", "sharedframework", "dotnet-sharedframework-debian_config.json");
```

Comment:
Can we make a Dirs property for this folder? `packaging\deb-package'  Or at least some sort of common place?